packages/shopping/src/__tests__/acceptance/authorization.acceptance.ts

```diff
@@ -132,19 +131,8 @@ describe('authorization', function (this: Suite) {
   }
 
   async function createAUser(userData: object) {
-    const encryptedPassword = await passwordHasher.hashPassword(userPassword);
-    const aUser = await userRepo.create(userData);
-
-    // MongoDB returns an id object we need to convert to string
-    aUser.id = aUser.id.toString();
-
-    await userRepo.userCredentials(aUser.id).create({
-      password: encryptedPassword,
-    });
-    return aUser;
-  }
-
-  async function createPasswordHasher() {
-    passwordHasher = await app.get(PasswordHasherBindings.PASSWORD_HASHER);
+    const aNewUser = new NewUser(userData);
```

Comment:
`aNewUser` -> simply `user`? If we can use more sensible names, we should.

Reply:
@hacksparrow I didn't want to change the convention that was already in place since this was more of a refactoring task. I can do that now. All these names were already in place before this change

Reply:
Feel free to make improvements wherever applicable.

Reply:
Cool